themes/opentermsarchive/assets/js/languageSwitcher.js

```diff
@@ -0,0 +1,10 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const $languageSwitcher = document.querySelector('.language-switcher');
+
+  document.querySelectorAll('.language-switcher--current, .language-switcher__item').forEach($el => {
+    $el.addEventListener('click', () => {
+      console.log('azdazd');
```

Comment:
Are you sure about that `azdazd`? 😄

Reply:
That's a good one! :)

I've corrected and modified the JavaScript linter so that it doesn't allow console logs.